django/contrib/admin/widgets.py

```diff
@@ -410,6 +410,7 @@ def build_attrs(self, base_attrs, extra_attrs=None):
         attrs.setdefault('class', '')
         attrs.update({
             'data-ajax--cache': 'true',
+            'data-ajax--delay': 500,
```

Comment:
I think we should use **250** by default (see [Select2 documentation](https://select2.org/data-sources/ajax#rate-limiting-requests)). As you mentioned in the ticket, users can always increase this value locally or combine it with the [minimumInputLength](https://select2.org/searching#minimum-search-term-length) option.

Reply:
Yep, I think 500 is massive™ delay. Let's try 250.